### PR TITLE
`treehouses clone reboot`

### PR DESCRIPTION
**Pull request by XavierElon1**

### CODE REVIEW

File: modules/clone.sh [hunk 1]
```diff
@@ -42,5 +71,11 @@ function clone_help {
   echo "      Will clone the current system to /dev/sdb (by default)."
   echo ""
   echo "  $(basename "$0") clone /dev/sda"
-  echo "      Will clone the current system to /dev/sda"
+  echo "      Will clone the current system to /dev/sda."
+  echo ""
+  echo "  $(basename "$@") clone --reboot"
```
Comment (kjong) [inline modules/clone.sh]: `echo "  $(basename "$@") clone --reboot"` should be
`echo "  $(basename "$0") clone --reboot"`?

Same thing with line 79.

Comment (xavierh93) [reply modules/clone.sh]: Fixed

**Comment by dogi** [comment]
@xavierh93 no tabs and no trailing tabs

File: modules/clone.sh [hunk 2]
```diff
@@ -1,7 +1,31 @@
...
+      exit 1
+     fi
+  elif [ $1 ]; then
+    if [ $1 = "--reboot" ]; then	
+      echo "reboot set"
+      reboot=true
+    else
+       device=$1
+    fi
+  fi
```
Comment (dogi) [inline modules/clone.sh]: @xavierh93 wonder if we can express this more elegant ;)

**Pull request closed without merging**